includes/reader-revenue/stripe/class-stripe-webhooks.php

```diff
@@ -0,0 +1,440 @@
+<?php
+/**
+ * Stripe Webhooks.
+ *
+ * @package Newspack
+ */
+
+// phpcs:disable WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Stripe Webhooks.
+ */
+class Stripe_Webhooks extends Stripe_Connection {
```

Comment:
Although they are related, I don't think `Stripe_Webhooks` should extend `Stripe_Connection`. What are the benefits of an instance of `Stripe_Webhooks` inheriting these methods?

Reply:
I agree, this looks weird. Changed in 327358f03944851b47e4d657817938c2b0e0762c